ixbrlparse/components/transform.py

```diff
@@ -1,14 +1,18 @@
 from copy import deepcopy
 from typing import List, Optional, Type, Union
 
+from .functionixt import ixtNamespaceFunctions
+
 
 class ixbrlFormat:
     def __init__(
         self,
+        numeric: bool,
```

Comment:
For backwards compatability, maybe move to the end (before `ixt` and give default `= True`)